lib/grp.gi

```diff
@@ -221,13 +223,13 @@ BindGlobal( "IS_PGROUP_FROM_SIZE",
       return true;
     elif s = infinity then
```

Comment:
This code is correct as long as ````IsPgroup```` implies ````IsFinite````. If that will change, this will suggest that the function is not working properly if the group is infinite. Fortunately enough, you only call this function for ````IsFinite```` groups later on in the ````IsPgroup```` code. Anyway, could you maybe add some comment here to reflect this fact that two years from now someone would not be confused by this? 

Or maybe it would be better to rename the function to ````IS_PGROUP_FROM_SIZE_IF_FINITE````, and return ````fail```` here, instead of ````false````

Reply:
I agree that returning `fail` here is superior.